packages/@okta/vuepress-site/docs/reference/registration-hook/index.md

```diff
@@ -198,141 +198,216 @@ Registrations are allowed by default, so setting a value of `ALLOW` for the `act
 
 See [error](/docs/concepts/inline-hooks/#error) for general information about the structure to use for the `error` object.
 
-For the Registration Inline Hook, the `error` object provides a way of displaying an error message to the end user who is trying to register.
+For the Registration Inline Hook, the `error` object provides a way of displaying an error message to the end user who is trying to register or update their profile.
+
+> **Note:** Generic Progressive Enrollment error messages are only available in the Identity Engine.
 
 * If you're using the Okta Sign-In Widget for Profile Enrollment, only the `errorSummary` messages of the `errorCauses` objects that your external service returns appear as inline errors, given the following:
 
    * You don't customize the error handling behavior of the widget.
-   * The `location` of `errorSummary` in the `errorCauses` object specifies the request object's user profile attribute. See [JSON response payload objects - error](/docs/concepts/inline-hooks/#error).
+   * The `location` of `errorSummary` in the `errorCauses` object specifies the request object's user profile attribute.
```

Comment:
I question this bullet point. Namely, even if it's required. I'd need to do some testing, but leaving this attribute blank, I believe, will not have an affect on where the UI message appears. Or, can error messages be tied, inline, to the appropriate progressive profiling field? The `location` field is added from the developer as a string field to aid in troubleshooting, according to our doc. https://developer.okta.com/docs/concepts/inline-hooks/#error

Reply:
@ericalong-okta We worked on this together. What are your thoughts?

Reply:
@brianduffield-okta it'll affect the UI error message if the `location` attribute is left blank, the improvement was requested by [OKTA-473152](https://oktainc.atlassian.net/browse/OKTA-473152). The error messages are now tied and inline to SIW fields for both registration (v1), SSR and PE (oie).